docs/source/getting_started/changelog.rst

```diff
@@ -3,16 +3,301 @@
 JupyterLab Changelog
 ====================
 
-v1.0.0 (not released yet)
--------------------------
+v1.0.0 (pre release)
+--------------------
 
-2019
-^^^^
 
 See the `JupyterLab
 1.0.0 <https://github.com/jupyterlab/jupyterlab/milestone/2?closed=1>`__
 milestone on GitHub for the full list of pull requests and issues closed.
 
+
+* Make The Defaultsession Own (Create And Dispose) Its Kernel Connection. (`#6503 <https://github.com/jupyterlab/jupyterlab/pull/6503>`__, `#6142 <https://github.com/jupyterlab/jupyterlab/issues/6142>`__)
+
+
+Lab config:
+
+* Lab Config Urls (`#6585 <https://github.com/jupyterlab/jupyterlab/pull/6585>`__, `#6584 <https://github.com/jupyterlab/jupyterlab/issues/6584>`__)
+* Update For Changes To Server (`#6509 <https://github.com/jupyterlab/jupyterlab/pull/6509>`__, `#6508 <https://github.com/jupyterlab/jupyterlab/issues/6508>`__)
+
+
+??? breaking buttons
+* Use Jupyterlab's Theme Variables For Ui Component Hover Style. (`#6524 <https://github.com/jupyterlab/jupyterlab/pull/6524>`__, `#6520 <https://github.com/jupyterlab/jupyterlab/issues/6520>`__)
+
+
+Workspaces:
+
+* Stop Using Base_url In Workspace Name. (`#6473 <https://github.com/jupyterlab/jupyterlab/pull/6473>`__, `#5977 <https://github.com/jupyterlab/jupyterlab/issues/5977>`__)
+* Allow Passing "-" As A File To Jupyter Workspaces Import (`#6276 <https://github.com/jupyterlab/jupyterlab/pull/6276>`__)
+* Update Workspaces To Automatically Resolve. (`#6234 <https://github.com/jupyterlab/jupyterlab/pull/6234>`__, `#6210 <https://github.com/jupyterlab/jupyterlab/issues/6210>`__)
+* Update Workspaces List Method To Conform To Idataconnector List Method. (`#5975 <https://github.com/jupyterlab/jupyterlab/pull/5975>`__)
+
+
+
+run in console:
+* Pass Metadata From Notebook To Console (`#6063 <https://github.com/jupyterlab/jupyterlab/pull/6063>`__)
```

Comment:
* Send complete statements instead of current lines when stepping through code in a cell (`#6515 <https://github.com/jupyterlab/jupyterlab/pull/6515>`__)